config/config.go

```diff
@@ -19,6 +19,8 @@ type Configuration struct {
 		ClientID     string `yaml:"client_id"`
 		ClientSecret string `yaml:"client_secret"`
 		ApiKey       string `yaml:"api_key"`
+		LogDir       string `yaml:"log_dir"`
+		LogFile       string `yaml:"log_file"`
```

Comment:
I noticed we join LogDir with LogFile at some point. What do you think about having this as a single config? We can remove LogDir and make LogFile the absolute path to the log location. Something like:

`log_file: /home/pankaj/spinup.log`

Reply:
good point. Less confusion 👍